### PR TITLE
 css homepage

### PR DESCRIPTION
**Pull request by francoo-trainer**
styles are added on homepage

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -1,50 +1,25 @@
+main {
```
Comment (Theweeksad) [inline css/homepage.css]: don't use tags, use a main class instead, e.g. (.main)

**Review by Theweeksad — CHANGES_REQUESTED** [review]
please fix main class

**Review by Theweeksad — APPROVED** [review]
nice work!!!

**Review by Theweeksad — APPROVED** [review]
all ready, I think all ready to merge

**Review by johanmendezb — APPROVED** [review]
good job

**Comment by devdevsamuel** [comment]
Please attach ss of the layaout
good job

**Pull request merged**